utils/migrate_template_csv_to_rule.py

```diff
@@ -0,0 +1,638 @@
+import argparse
+import csv
+import os
+import pprint
+import re
+
+from ssg.constants import product_directories
+
+
+def escape_path(path):
+    return re.sub('[-\./]', '_', path)
+
+def accounts_password_csv_to_dict(csv_line, csv_data):
+    accounts_password = {}
+    accounts_password["template"] = "package_installed"
+
+    variable = csv_line[0]
+    rule_id = f"accounts_password_pam_{variable}"
+
+    operation = csv_line[1]
+
+    # Only credit related variables allow negative values
+    sign = "-?" if variable.endswith("credit") else ""
+
+    accounts_password["VARIABLE"] = variable
+    accounts_password["OPERATION"] = operation
+    accounts_password["SIGN"] = sign
+    csv_data[rule_id] = accounts_password
+    return accounts_password
+
+def audit_rules_execution_csv_to_dict(csv_line, csv_data):
+    audit_rules_execution = {}
+    audit_rules_execution["template"] = "audit_rules_privileged_commands"
+
+    path = csv_line[0]
+    name = escape_path(os.path.basename(path))
+    rule_id = f"audit_rules_execution_{name}"
+
+    # create_audit_rules_execution.py escapes the '/' when generating the OVAL
+    # This is not actually necessary
+    audit_rules_execution["PATH"] = path
+    audit_rules_execution["NAME"] = name
+
+    audit_rules_execution["ID"] = f"audit_rules_execution_{name}"
+    audit_rules_execution["TITLE"] = f"Record Any Attempts to Run {name}"
```

Comment:
@yuumasato  Is it expected that the people will always fill this in when they write `rule.yml`? I think that isn't user friendly. Should #4809 contain similar logic to fill it the id and title during the build?

Reply:
I agree, I have this already. I didn't push the next commits yet.

Reply:
@yuumasato So it's expected that the people will always need to fill the ID and title or it isn't expected? If the ID and title should be derived by the build system, I need to add that into #4809 . Actually I planned to do that, but then I saw this, so I'm asking about if it's needed. I would like to prevent duplicate work.

Reply:
> So it's expected that the people will always need to fill the ID and title or it isn't expected?

It is not expected that people will write the ID.

Reply:
ok thanks